HUD/EndTextCommandIsThisHelpMessageBeingDisplayed.md

```diff
@@ -6,11 +6,12 @@ aliases: ["0x10BDDBFC529428DD"]
 
 ```c
 // 0x10BDDBFC529428DD 0xF63A13EC
-BOOL END_TEXT_COMMAND_IS_THIS_HELP_MESSAGE_BEING_DISPLAYED(int p0);
+BOOL END_TEXT_COMMAND_IS_THIS_HELP_MESSAGE_BEING_DISPLAYED(int hudIndex);
 ```
 
 
 ## Parameters
-* **p0**: 
+* **hudIndex**: The hud index for help message. `0` is `HELP_TEXT`, `1` and `2` is `FLOATING_HELP_TEXT_1` and `FLOATING_HELP_TEXT_2`.
```

Comment:
Maybe declare an enum for the `hudIndex` under the description? (by line 11)

i.e.

```c
enum HudIndexes {
    HELP_TEXT = 0,
    FLOATING_HELP_TEXT_1 = 1,
    FLOATING_HELP_TEXT_2 = 2,
}
```

It's neater in my opinion, just nitpicking really 😃

Reply:
I will add it now, thanks!